docs/manual/library/games/metadata.md

```diff
@@ -0,0 +1,69 @@
+## Metadata
```

Comment:
This page should definitely include two things that have to be commonly addressed:

- some metadata are missing, what to do. This is usually cases where people restart Playnite after first time startup and don't let everything download
- not all metadata sources or library plugins can provide information for all fields. Typical case are icons.

Basically somehow integrate information from Jeshibu's page about this:

https://github.com/Jeshibu/PlayniteExtensionsFeatures/blob/main/faq.md#my-games-are-missing-covers-or-other-data-or-what-is-metadata

Reply:
https://github.com/JosefNemec/PlayniteDocs/pull/1/commits/763d09a4cd286a6f17ea3f569f1c5bec3058ed46